src/Modules/AdminDashboard/Componenets/Dashboard/Dashboard.tsx

```diff
@@ -1,3 +1,249 @@
-export default function Dashboard() {
-  return <div>Home</div>;
+import { Box, Stack, Typography } from "@mui/material";
+import axios from "axios";
+import { useCallback, useEffect, useState } from "react";
+import { useAuth } from "../../../../Context/AuthContext/AuthContext";
+import { Charts } from "../../../../Interfaces/interFaces";
+import { PieChart } from "@mui/x-charts/PieChart";
+import AddHomeWorkIcon from "@mui/icons-material/AddHomeWork";
+import Shop2Icon from "@mui/icons-material/Shop2";
+import CampaignIcon from "@mui/icons-material/Campaign";
+import { toast } from "react-toastify";
+import { getBaseUrl } from "../../../../Utils/Utils";
+import { BarChart } from "@mui/x-charts";
+
+export default function DashboardHome() {
+  const [chartsData, setChartsData] = useState<Charts>({});
+
+  const { requestHeaders } = useAuth();
+
+  const getCharts = useCallback(async () => {
+    try {
+      const response = await axios.get(
+        `${getBaseUrl()}/api/v0/admin/dashboard`,
+        {
+          headers: requestHeaders,
+        }
+      );
+      setChartsData(response?.data?.data);
+    } catch (error) {
+      if (axios.isAxiosError(error) && error.response) {
+        toast.error(error.response.data.message || "fail signin");
+      }
+    }
+  }, [requestHeaders]);
+
+  useEffect(() => {
+    getCharts();
+  }, [getCharts]);
+
+  return (
+    <Stack
+      component={"section"}
+      width="100%"
+      display="flex"
+      direction="column"
+      justifyContent="flex-start"
+      gap={12}
+      marginTop={8}
+    >
+      <Stack
+        gap="20px"
+        alignItems="center"
+        justifyContent="space-around"
+        direction="row"
+        sx={{
+          flexWrap: "wrap",
+        }}
+      >
+        <Stack
+          spacing={3}
+          alignItems="center"
+          justifyContent="space-around"
+          direction="row"
+          margin="auto !important"
+          sx={{
+            bgcolor: "rgba(26, 27, 30, 1)",
```

Comment:
Using hardcoded colors like this will make us need to handle light and dark colors for them manually. On the other hand, if we stick to the MUI colors, the theme colors will change automatically without needing to exert any effort in managing the colors.

Reply:
The same applies to spacing (paddings and margins), padding: 4 in the sx prop will resolve to 4*8 because the spacing point is equivalent to 8 in MUI's current design. This will make it easier to update the spacing later.